File: core/src/main/java/com/loitpcore/views/piano/listener/LoadAudioMessage.java
package com.loitpcore.views.piano.listener;

/**
 * Created by Loitp on 27.09.2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
public interface LoadAudioMessage {
    void sendStartMessage();

    void sendFinishMessage();

    void sendErrorMessage(Exception e);

    void sendProgressMessage(int progress);
}

File: core/src/main/java/com/loitp/views/piano/listener/LoadAudioMessage.kt
package com.loitp.views.piano.listener

/**
 * Created by Loitp on 27.09.2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
interface LoadAudioMessage {
    fun sendStartMessage()
    fun sendFinishMessage()
    fun sendErrorMessage(e: Exception?)
    fun sendProgressMessage(progress: Int)
}